03-Node-npm-modules/movie.js

```diff
@@ -0,0 +1,27 @@
+var director = require('./director.js')
```

Comment:
Also if this is a class (like you exported on the director.js file, this should be Director instead